Project-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,37 @@
 import React from 'react';
 import './Button.css';
+
+const NumberButton = props => {
+  return (
+    <div className="number-button">
+        <div className = "row clear-row">
+            <span class="button" id="clear">clear</span>
+            <span class="operator button" id="times">&divide;</span>
+        </div>
+        <div className = "row top-row">
+            <span class="number button" id="seven">7</span>
+            <span class="number button" id="eight">8</span>
+            <span class="number button" id="nine">9</span>
+            <span class="operator button" id="times">&times;</span>
+        </div>
+        <div className = "row middle-row">
+            <span class="number button" id="four">4</span>
+            <span class="number button" id="five">5</span>
+            <span class="number button" id="six">6</span>
+            <span class="operator button" id="minus">-</span>
+        </div>        
+        <div className = "row bottom-row">
+            <span class="number button" id="one">1</span>
+            <span class="number button" id="two">2</span>
+            <span class="number button" id="three">#</span>
+            <span class="operator button" id="minus">+</span>
+        </div>
+        <div className = "row zero-row">
+            <span class="number button" id="zero">0</span>
+            <span class="operator button" id="equals">=</span>
```

Comment:
Calculator looks great! For some reason the compiler was able to pass these classes down using "class" instead of "className". It still throws an error in the console, so it's still best practice to use "className" with JSX. 

```suggestion
            <span className="operator button" id="equals">=</span>
```

![screen shot 2019-02-13 at 4 54 35 pm](https://user-images.githubusercontent.com/42302710/52749865-18e63380-2fb0-11e9-899c-e8fe9e46f884.png)

https://reactjs.org/blog/2017/09/08/dom-attributes-in-react-16.html